crates/rolldown/src/module_finalizers/scope_hoisting/mod.rs

```diff
@@ -86,6 +86,11 @@ impl<'me, 'ast> ScopeHoistingFinalizer<'me, 'ast> {
     symbol_ref: SymbolRef,
     preserve_this_semantic_if_needed: bool,
   ) -> ast::Expression<'ast> {
+    if !symbol_ref.is_declared_in_root_scope(self.ctx.symbol_db) {
+      // No fancy things on scoped symbols.
```

Comment:
root scope symbol also has scope, the `scoped symbol` sounds a little ambiguous, maybe `No fancy things on none root scope symbols` ?

Reply:
Done